.github/workflows/reusable/cached-build/action.yml

```diff
@@ -35,5 +35,7 @@ runs:
       shell: bash
     - name: build
       if: steps.cache-build.outputs.cache-hit != 'true'
-      run: npm run build
+      run: npm run build && npm ci --no-audit
```

Comment:
Add comment as to why this is necessary?

Reply:
This is a known bug in npm, another npm ci after build is needed to install the binaries generated by submodules

Reply:
@ptesavol Which binaries weren't installed in practice? Could you provide a link to that bug?

Reply:
streamr-keyserver was not installed. https://github.com/npm/cli/issues/4591